src/app/saga.ts

```diff
@@ -430,97 +415,6 @@ export function* handleSetAppState(action: SetAppState) {
   }
 }
 
-export function* runCentralPhoneVerificationMigration() {
-  const shouldRunVerificationMigration = yield* select(shouldRunVerificationMigrationSelector)
-  if (!shouldRunVerificationMigration) {
-    return
-  }
-
-  const privateDataEncryptionKey = yield* select(dataEncryptionKeySelector)
-  if (!privateDataEncryptionKey) {
-    Logger.warn(
-      `${TAG}@runCentralPhoneVerificationMigration`,
-      'No data encryption key was found in the store. This should never happen.'
-    )
-    return
-  }
-
-  const address = yield* select(walletAddressSelector)
-  const mtwAddress = yield* select(mtwAddressSelector)
-  const phoneNumber = yield* select(e164NumberSelector)
-  const publicDataEncryptionKey = compressedPubKey(hexToBuffer(privateDataEncryptionKey))
-
-  try {
-    const signedMessage = yield* call(retrieveSignedMessage)
-    if (!signedMessage) {
-      Logger.warn(
-        `${TAG}@runCentralPhoneVerificationMigration`,
-        'No signed message was found for this user. Skipping CPV migration.'
-      )
-      return
-    }
-    if (!phoneNumber) {
-      Logger.warn(
-        `${TAG}@runCentralPhoneVerificationMigration`,
-        'No phone number was found for this user. Skipping CPV migration.'
-      )
-      return
-    }
-
-    const saltCache: E164NumberToSaltType = yield* select(e164NumberToSaltSelector)
-    const cachedSalt = saltCache[phoneNumber]
-    if (!cachedSalt) {
-      Logger.warn(
-        `${TAG}@runCentralPhoneVerificationMigration`,
-        'No salt was cached for phone number. Skipping CPV migration.'
-      )
-      return
-    }
-
-    Logger.debug(
-      `${TAG}@runCentralPhoneVerificationMigration`,
-      'Starting to run central phone verification migration'
-    )
-
-    const phoneHash = yield* call(getPhoneHash, phoneNumber, cachedSalt)
-    const inviterAddress = yield* select(inviterAddressSelector)
-
-    const response = yield* call(fetch, networkConfig.migratePhoneVerificationUrl, {
```

Comment:
This mean we can remove the `migrateASv1Verification` endpoint in `identity-service`.